.github/workflows/test.yaml

```diff
@@ -1,120 +0,0 @@
-name: CI / Console
-
-on:
-  push:
-    branches: [ master ]
```

Comment:
should we be removing this file?

Reply:
oh wait, good catch. I was testing crd copy make target to make sure it works in GH action and must have accidentally removed this one.